frame.go

```diff
@@ -1503,18 +1523,21 @@ func (f *framer) writeQueryParams(opts *queryParams) {
 		}
 	}
 
-	if opts.keyspace != "" {
-		if f.proto > protoVersion4 {
+	// protoV5 specific things
+	if f.proto > protoVersion4 {
+		if opts.keyspace != "" {
 			flags |= flagWithKeyspace
-		} else {
-			panic(fmt.Errorf("the keyspace can only be set with protocol 5 or higher"))
```

Comment:
We should keep this panic (or return an error) if the user tries to set the keyspace but protocol version is not > 4

Reply:
Now I'm looking at it and releasing that that was a bit greedy decision to get rid of those panics. I'll change this one too

Reply:
And this one got changed.